src/main/java/org/zeromq/ZBeacon.java

```diff
@@ -11,20 +11,21 @@
 import java.nio.channels.ClosedChannelException;
 import java.nio.channels.DatagramChannel;
 import java.util.Arrays;
+import java.util.Objects;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.concurrent.atomic.AtomicReference;
 
 public class ZBeacon
 {
-    public static final long   DEFAULT_BROADCAST_INTERVAL = 1000L;
-    public static final String DEFAULT_BROADCAST_HOST     = "255.255.255.255";
-
-    private final int             port;
-    private InetAddress           broadcastInetAddress;
-    private final BroadcastClient broadcastClient;
-    private final BroadcastServer broadcastServer;
-    private final byte[]          beacon;
-    private byte[]                prefix            = {};
-    private long                  broadcastInterval = DEFAULT_BROADCAST_INTERVAL;
-    private Listener              listener          = null;
+    public static final long    DEFAULT_BROADCAST_INTERVAL = 1000L;
+    public static final String  DEFAULT_BROADCAST_HOST     = "255.255.255.255";
```

Comment:
How that does handle IPv6 ?

Reply:
Default is IPv4 format, users wanting to use IPv6 can do it by configuring the client host; using one of the constructors ZBeacon(String host, ...), or the new builder.